tests/integration/Sync_test.cpp

```diff
@@ -2049,7 +2049,7 @@ void waitonsyncs(chrono::seconds d = std::chrono::seconds(4), StandardClient* c1
     auto totalTimeoutStart = chrono::steady_clock::now();
     auto start = chrono::steady_clock::now();
     std::vector<StandardClient*> v{ c1, c2, c3, c4 };
-    bool onelastsyncdown = true;
+//    bool onelastsyncdown = true;
```

Comment:
IMHO, if the commented code in this test is not going to be restored, better to remove it.
Otherwise, it would be good to add a comment on why it's commented.

Reply:
Ok, removed.   I wasn't sure if there were still cases where it might be needed but the tests are passing so let's remove it.